content/en/docs/concepts/observability-primer.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Observability Primer
+weight: 9
+---
+
+Before digging into OpenTelemetry, it is important to understand some core
+concepts first.
+
+## What is Observability?
+
+Observability lets us understand a system from the outside, by letting us ask
+questions about that system without knowing its inner workings. Furthermore,
+allows us to easily troubleshoot and handle novel problems (i.e. “unknown
+unknowns”), and helps us answer the question, "Why is this happening?"
+
+In order to be able to ask those questions of a system, the application must be
+properly instrumented. That is, the application code must emit
+[traces](/docs/concepts/observability-primer/#distributed-traces),
+[metrics](/docs/concepts/observability-primer/#reliability--metrics),
+[logs](/docs/concepts/observability-primer/#logs). An application is properly
+instrumented when developers don't need to add more instrumentation to
+troubleshoot an issue, because they have all of the information they need.
+
+[**OpenTelemetry**](/docs/concepts/what-is-opentelemetry) is the mechanism by
+which application code is instrumented, to help make a system observable.
+
+## Reliability & Metrics
+
+**Telemetry** refers to data emitted from a system, about its behaviour. The
+data can come in the form of [Traces](#distributed-traces),
+[Metrics](#reliability--metrics), and [Logs](#logs).
+
+**Reliability** answers the question: “Is the service doing what users expect it
+to be doing?” A system could be up 100% of the time, but if, when a user clicks
+“Add to Cart” to add a black pair of pants to their shopping card, and instead,
+the system keeps adding a red pair of pants, then the system would be said to be
+**un**reliable.
+
+**Metrics** are aggregations over a period of time of numeric data about your
+infrastructure or application. Examples include: system error rate, CPU
+utilization, request rate for a given service.
+
+**SLI**, or Service Level Indicator, represents an item that is being measured.
+Although SLI created off of [metrics](#reliability--metrics), these metrics are
```

Comment:
Not necessarily. For example, with Honeycomb, SLIs are a computed expression with one or more attributes on a trace span as input (technically not just a trace span, but this is the most common use case). I don't think most vendors do this today, but it's entirely possible that more will do this in the future. This starts to tread into "what observability backends do" territory and I do worry that if we start documenting that stuff we could quickly come off as favoring one backend over another.

Reply:
Fair point. I've tweaked it to:

> **SLI**, or Service Level Indicator, represents a measurement of a service's behaviour. A good SLI measures your service from the perspective of your users. An example SLI can be the speed at which a web page loads.